components/post_view/post_message_view/post_message_view.jsx

```diff
@@ -187,6 +196,11 @@ export default class PostMessageView extends React.PureComponent {
                     />
                 </div>
                 {this.renderEditedIndicator()}
+                <Pluggable
+                    pluggableName='PostMessage'
+                    postId={post.id}
+                    onHeightChange={this.handleHeightReceived}
+                />
```

Comment:
Can we name this something more descriptive? Maybe like `PostMessageAttachment`? I want it to be clear that this is something being added after the post message, not instead of

Reply:
Also why pass just the post ID? Wouldn't it be better for the plugin to get the whole post?

Reply:
Actually, passing the whole post is my initial preference and it's convenient when immediately available since plugin developers don't need to deal with the store at first. 0/5 but I'm with `postId` only for consistency since we're doing that in other place like: https://github.com/mattermost/mattermost-webapp/blob/7e2bf77e6ff28ea9221c1dfc3b5273dda540555c/components/dot_menu/dot_menu.jsx#L308-L311

Reply:
@jwilander I didn't change passing the post ID.  Let me know your thought about it.  I'll change it if you think it's the way we should do it.  Thanks!